### ISSUE TITLE
How can we get the model checkpoints to test it out on our own data?

### ISSUE THREAD

**Issue by RogerPallares** [aliciachenw/PIPsUS#2]
Hi, I was wondering how I could download the checkpoints for PIPsUS, as I would like to try it on my data!

Thanks!

**Comment by aliciachenw** [comment]
Hi Roger, thanks for your interest! The link to the open model weight is added in the readme file.

**Comment by malakiss** [comment]
is there any steps needed to start finetuning on my data?

**Comment by aliciachenw** [comment]
> is there any steps needed to start finetuning on my data?

Hi Malak @malakiss ,

Thank you for your interest in this work! I will suggest referring to the script miccai_main_finetune_pipsUS_echo.py to learn about the training loop and how to generate pseudo ground truth using pips++ (in the function generate_pseudo_gt_pips2). Of course, you need to write your own dataset class for the sequences.

Best,
Wanwen

**Comment by malakiss** [comment]
Thank you !
but could you provide the  pseudogrondtruth 
if os.name == 'nt':
    PSEUDO_LABEL_PATH = 'D:/Wanwen/EchoNet/echonetdynamic-2/EchoNet-Dynamic/pseudo20/'
else:
    PSEUDO_LABEL_PATH = '/workspace/us_seq_dataset/EchoNet/echonetdynamic-2/EchoNet-Dynamic/EchoNet-Dynamic/pseudo20/'

**Comment by aliciachenw** [comment]
Hi, if you hope to finetune the model on your own data, you need to use PIPs++ to generate pseudo ground truth.

**Comment by malakiss** [comment]
then finetune without warmup ? 
and how how to infere if you could provide the script i would be grateful

**Comment by aliciachenw** [comment]
You could do a warm-up training, depending on how you would like to approach your use case. We provided a pre-trained model on EchoNet in the README if you hope to fine-tune on top of it.
Sample inference codes are in the script:  inference_pipsUS.py.
